modules/menu/menu.c

```diff
@@ -304,21 +334,38 @@ static void play_ringback(const struct call *call)
 }
 
 
-static void play_resume(const struct call *closed)
+static void check_ringback(struct call *call)
+{
+	bool ring;
+	enum sdp_dir ardir;
+
+	ardir = sdp_media_rdir(stream_sdpmedia(audio_strm(call_audio(call))));
+	ring = (ardir & SDP_RECVONLY) ? false : true;
```

Comment:
`ardir` and `ring` can be declared and set at the same time

also:

`ring = !(ardir & SDP_RECVONLY)`